src/ddspdrum/module.py

```diff
@@ -353,10 +350,14 @@ def __init__(
         sustain_duration: float,
         pitch_adsr: ADSR = ADSR(),
         amp_adsr: ADSR = ADSR(),
-        vco: VCO = VCO(),
+        vco: Union[VCO, List[VCO]] = VCO(),
```

Comment:
Can we just force it to be a list of VCOs and call the variable vcos? This Union this is kinda weird.

Reply:
The idea is that you could just feed it a single VCO and it would work fine. It's an interface choice. But as you say, might be better to work with a fixed architecture anyway. Thoughts?

Reply:
I don't really have good insight on how many oscillators is enough :) Your call with @jorshi

Reply:
Fixing with two for now sounds good. When we were starting this project I imagined that we would have a collection of building block dsp modules, and then some preset architectures based on existing drum machines. Like an Arp Axxe architecture, SH101 architecture, etc. As well as the ability for end users to specify and learn on their own architectures.